### PR TITLE
[9.0] document_page_environmental_aspect

### PR DESCRIPTION
**Pull request by naousse**

### CODE REVIEW

File: document_page_environmental_aspect/README.rst [hunk 1]
```diff
@@ -0,0 +1,73 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :alt: License
+
+Document Management - Wiki - Environmental Aspects
+==================================================
+
+This module was written to extend the functionality of Environment Management System. Add a document template for Environmental Aspects.
```
Comment (elicoidal) [inline document_page_environmental_aspect/README.rst]: This module extends the functionality of Environment Management System and add a document template

File: document_page_environmental_aspect/README.rst [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+
+Installation
+============
+
+To install this module, you need to:
+
+ * Clone the project from git@github.com:OCA/knowledge.git
```
Comment (elicoidal) [inline document_page_environmental_aspect/README.rst]: not necessary (2 lines)

File: document_page_environmental_aspect/README.rst [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+ * Clone the project from git@github.com:OCA/management-system.git
+ * Add both projects to the addons path
+
+Configuration
+=============
+
+To configure this module, you need to:
```
Comment (elicoidal) [inline document_page_environmental_aspect/README.rst:19]: No configuration required.

**Comment by max3903** [comment]
:+1:

**Comment by dreispt** [comment]
:+1:

**Pull request merged**